provider/server/machine_create_test.go

```diff
@@ -400,7 +400,7 @@ var _ = Describe("CreateMachine", func() {
 			HaveField("Machine.Spec.NetworkInterfaces", ContainElement(&iri.NetworkInterface{
 				Name: "nic-1",
 			})),
-			HaveField("Machine.Status.ObservedGeneration", BeZero()),
+			HaveField("Machine.Status.ObservedGeneration", Equal(int64(1))),
```

Comment:
FYI (pls don't solve it in this PR): i checked logic and it looks little weird for me, we return api.Machine.Metadata.Generation as observedGeneration. But it isn't true because worker can be overloaded  or stuck somewhere and we don't have guaranty, if desired state state was processed. So technically we are reporting invalid information to customer.

Reply:
Reverted as no longer needed